interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -0,0 +1,48 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <b>Experiments with the highest range: </b>{{experiments.length}}
+  </h4>
+  <button class="btn btn-primary" ng-click="showTop()"
+          ng-hide="topMode">Show Top {{topNum}}</button>
+  <button class="btn btn-primary" ng-click="showAll()"
+          ng-show="topMode">Show All</button>
+
+  <br />
+  (Click
+  <i class="fa fa-plus-square fa-lg" aria-hidden="true"></i> or
```

Comment:
![tagihqdqctlg](https://cloud.githubusercontent.com/assets/542643/21432031/b3500d84-c837-11e6-9cf2-16d908d000bd.png)
I made a quick sketchboard with my concerns about the buttonizing + thoughts.